functions/src/import-votes.ts

```diff
@@ -0,0 +1,56 @@
+import * as functions from "firebase-functions";
+import * as firebase from "firebase-admin";
+
+import { Config } from "./config";
+import { Vote } from "./cast-vote";
+
+const config = functions.config() as Config;
+const importVotesConfigs = config.features.import_votes;
+const db = firebase.firestore();
+
+const fromJSONtoVote = (JSONString: string) => {
+  const supposedlyValidVotes = JSON.parse(JSONString);
+  const validVotes: Vote[] = [];
+  for (const supposedlyValidVote of supposedlyValidVotes.votes) {
+    if (
+      "value" in supposedlyValidVote &&
+      "campaign" in supposedlyValidVote &&
+      "agency" in supposedlyValidVote
+    ) {
+      validVotes.push(supposedlyValidVote);
+    }
+  }
```

Comment:
That's a filter :) use `filter` for better clarity

Reply:
filter is way cooler indeed 😄